### ISSUE TITLE
Rename from Viz Preview

### ISSUE THREAD

**Issue by curran** [datavis-tech/vizhub-issue-tracker#327]

**Comment by stushurik** [comment]
@curran maybe it would also helpful to allow edit viz title and description from viz viewer not only via index.html and readme.md.
when i first saw vizhubi was very frustrated how to edit it and constantly tried to click on  viewer's sections.

**Comment by curran** [comment]
It's a great idea! I agree the current behavior is not intuitive.

I wonder how hard it would be to make the title itself editable in the viz viewer page.

It would need to change the title in `index.html` as well once it updates, but that should be doable maybe with regex.

**Comment by stushurik** [comment]
yes I suggest make it bi-directional, so if user changes title it updates index.html and vice versa

**Comment by curran** [comment]
Agreed.
